terraso_backend/apps/graphql/schema/shared_data.py

```diff
@@ -0,0 +1,89 @@
+import graphene
+import structlog
+from graphene import relay
+from graphene_django import DjangoObjectType
+
+from apps.graphql.exceptions import GraphQLNotAllowedException
+from apps.shared_data.models import DataEntry
+
+from .commons import BaseDeleteMutation, BaseWriteMutation, TerrasoConnection
+from .constants import MutationTypes
+
+logger = structlog.get_logger(__name__)
+
+
+class DataEntryNode(DjangoObjectType):
+    id = graphene.ID(source="pk", required=True)
+
+    class Meta:
+        model = DataEntry
+        filter_fields = {
+            "name": ["icontains"],
+            "description": ["icontains"],
+            "slug": ["exact", "icontains"],
+            "url": ["icontains"],
+            "groups__slug": ["icontains"],
```

Comment:
question: should this match `slug`?

Reply:
This adds a filter that matches any part of the slug ignoring the word case.